src/patterns/data/default/blood_vessel_pattern.tsv

```diff
@@ -0,0 +1,56 @@
+defined_class	label	parent	part_of_vccf	definition	xref_vccf	synonym	synonym_xrefs	connecting_branch_of	tributary_of	vessel_supplies_blood_to	vessel_drains_blood_from	creator									
+UBERON:8600051	segmental pulmonary vein	UBERON:0001638	UBERON:0002048	A vein that is tributary of the main pulmonary veins and drains oxygenated blood from a segment of the lung.	https://radiologykey.com/6-pulmonary-artery-and-vein/	Segmental pulmonary vein	FMA:9411		UBERON:0002016		UBERON:8600017	https://orcid.org/0000-0001-6677-8489 | https://orcid.org/0000-0002-2597-881X | https://orcid.org/0000-0002-5574-4272									
+UBERON:8600052	subsegmental pulmonary vein	UBERON:0001638	UBERON:0002048	A vein that is tributary of the segmental pulmonary veins and drains oxygenated blood from a subsegment of the lung.	https://radiologykey.com/6-pulmonary-artery-and-vein/	Subsegmental pulmonary vein	FMA:9413		UBERON:8600051		UBERON:8600017	https://orcid.org/0000-0001-6677-8489 | https://orcid.org/0000-0002-2597-881X | https://orcid.org/0000-0002-5574-4272									
+UBERON:8600053	esophageal vein	UBERON:0001638	UBERON:0000974	A vein that is tributary of the azygos vein and drains blood from the esophagus.	wikipedia:Esophageal_veins	Esophageal vein	FMA:4748		UBERON:0001594		UBERON:0001043	https://orcid.org/0000-0001-6677-8489 | https://orcid.org/0000-0002-2597-881X | https://orcid.org/0000-0002-5574-4272									

```

Comment:
It is highly untypical to have a dosdp pattern with optional columns (i.e. where you have empty values for, say `connecting_branch_of`). This sounds more like ROBOT template?

Reply:
On second thought, it does not look wrong (you are sometimes adding existentials and other times not) but I would recommend to ask @dosumis or @balhoff for a review as I dont remember if this is in line with the ideas of a pattern (having lots of optional fields).

Reply:
We do support optionals (added a couple of years ago).